src/main.js

```diff
@@ -435,7 +435,7 @@ Formsy.Form = React.createClass({
   render: function () {
 
     return (
-      <form {...this.props} onSubmit={this.submit}>
+      <form id={this.props.id} {...this.props} onSubmit={this.submit}>
```

Comment:
I thought so, however, the version I downloaded for use in one of my projects does not. Any ideas why?